### PR TITLE
Using rlog in agent and tapper & removing unreferenced code

### PR DESCRIPTION
**Pull request by IgorGov**

### CODE REVIEW

**Review by gadotroee — APPROVED** [review]

File: agent/main.go [hunk 1]
```diff
@@ -77,13 +78,13 @@ func main() {
...
 		tap.StartPassiveTapper(tapOpts, filteredOutputItemsChannel, extensions, filteringOptions)
-		socketConnection, err := shared.ConnectToSocketServer(*apiServerAddress, shared.DEFAULT_SOCKET_RETRIES, shared.DEFAULT_SOCKET_RETRY_SLEEP_TIME, false)
+		socketConnection, err := utils.ConnectToSocketServer(*apiServerAddress)
 		if err != nil {
 			panic(fmt.Sprintf("Error connecting to socket server at %s %v", *apiServerAddress, err))
 		}
+		rlog.Infof("Connected successfully to websocket %s", *apiServerAddress)
 
 		go pipeTapChannelToSocket(socketConnection, filteredOutputItemsChannel)
-		// go pipeOutboundLinksChannelToSocket(socketConnection, outboundLinkOutputChannel)
```
Comment (RoyUP9) [inline agent/main.go:86]: why is it commented?

File: agent/pkg/utils/utils.go [hunk 2]
```diff
@@ -54,15 +54,15 @@ func ReverseSlice(data interface{}) {
 
 func CheckErr(e error) {
 	if e != nil {
-		log.Printf("%v", e)
+		rlog.Infof("%v", e)
```
Comment (RoyUP9) [inline agent/pkg/utils/utils.go:57]: should this be error? and why is the panic commented?

File: agent/pkg/utils/utils.go [hunk 3]
```diff
@@ -54,15 +54,15 @@ func ReverseSlice(data interface{}) {
...
 		//panic(e)
 	}
 }
 
 func SetHostname(address, newHostname string) string {
 	replacedUrl, err := url.Parse(address)
-	if err != nil{
-		log.Printf("error replacing hostname to %s in address %s, returning original %v",newHostname, address, err)
+	if err != nil {
+		rlog.Infof("error replacing hostname to %s in address %s, returning original %v", newHostname, address, err)
```
Comment (RoyUP9) [inline agent/pkg/utils/utils.go:65]: should this be error?

File: agent/pkg/rules/rulesHTTP.go [hunk 4]
```diff
@@ -65,7 +66,7 @@ func MatchRequestPolicy(harEntry har.Entry, service string) []RulesMatched {
 				if err != nil {
 					continue
 				}
-				fmt.Println(matchValue, rule.Value)
+				rlog.Info(matchValue, rule.Value)
```
Comment (RoyUP9) [inline agent/pkg/rules/rulesHTTP.go:69]: maybe add a title that can explain this log

**Pull request merged**